__tests__/app.test.js

```diff
@@ -151,3 +154,57 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("POST:201 inserts a new comment to the comments table and sends the new comment back to the client", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "I like this article",
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment.comment_id).toBe(19);
+        expect(body.comment.article_id).toBe(2);
+      });
+  });
```

Comment:
Really good test description, but the test is not complete yet. We should be asserting for the entire comment object instead of just two properties. I would recommend looking into the .`toMatchObject` matcher for this as well. You have already asserting for the article_id which is really good, but we also know what the username and body should be as well and should also include all of the other properties we expect.